examples/templateservicebroker/templateservicebroker-template.yaml

```diff
@@ -138,3 +139,57 @@ objects:
   - kind: ServiceAccount
     namespace: ${NAMESPACE}
     name: apiserver
+
+# for the SA for the service catalog to use when calling the tsb
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    namespace: ${NAMESPACE}
+    name: templateservicebroker-client
+
+# allow the SC's SA to call the TSB
+- kind: ClusterRoleBinding
+  apiVersion: v1
+  metadata:
+    name: templateservicebroker-client
+  roleRef:
+    name: system:openshift:templateservicebroker-client
+  userNames:
```

Comment:
usernames are legacy (since 3.1 or so), use subjects

Reply:
Why not just use RBAC?

Reply:
because this PR isn't about "migrate from something that's working to something i'd have to learn"

Reply:
That's fair, but you are clearly fighting with the rough edges from the legacy APIs.  I will gladly help with the conversions :)